Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM ubuntu:24.04 AS build
 ENV TZ=UTC
 ENV LANG=C.UTF-8
 
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Add source for practicerom-dev install
+RUN apt-get update && apt-get install -y curl && \
+    curl -o /etc/apt/trusted.gpg.d/practicerom.gpg https://practicerom.com/public/packages/practicerom.gpg && \
+    echo 'deb [arch=amd64] http://practicerom.com/public/packages/debian unstable main' > /etc/apt/sources.list.d/practicerom.list
+
 # Install Required Dependencies
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
-    apt-get update && apt-get install -y \
-    curl \
+RUN apt-get update && apt-get install -y \
```

Comment:
I broke these into multiple steps because I think it is better organized this way.